src/scripts/scroll_to_bottom.js

```diff
@@ -83,6 +83,6 @@ export const clean = async function () {
   pageModifications.unregister(addButtonToPage);
   pageModifications.unregister(checkForButtonRemoved);
   stopScrolling();
-  scrollToBottomButton?.remove();
+  $(`#${scrollToBottomButtonId}`).remove();
```

Comment:
This only works if there is one stale button, which is a fair assumption most of the time, but not always. We can remove all stale buttons by using an attribute selector instead of an ID selector.

```suggestion
  $(`[id="${scrollToBottomButtonId}"]`).remove();
```

Reply:
Oh interesting! I assumed JQuery would passthrough to QSA, but clearly it doesn't (and in retrospect I was just reading the JQuery source and it's certainly not the kind of thing they would do :D) I accidentally did a sequence of events that made me think I had demonstrated that, too, but I think I had to throw a refresh in there.